.github/workflows/pre-commit.yml

```
@@ -0,0 +1,25 @@
+name: Pre-commit checks
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)